lib/command.rb

```diff
@@ -16,34 +17,36 @@ def build
   private
 
   def base_branch
-    config.fetch("base_branch", "origin/master")
+    config["base_branch"] || github_data.base_ref || "origin/master"
```

Comment:
`github.base_ref` will only be set on actions triggered by a pull request so we need to have a reasonable fallback here.